spec/mixlib/install/backend_spec.rb

```diff
@@ -43,7 +43,11 @@
   shared_examples_for "the right artifact info" do
     it "gives the right url artifact info" do
       if !expected_info.key?(:url)
-        expect(info.url).to match "https://"
+        if channel == :unstable
```

Comment:
This is not ideal. Maybe we can set the `expected_info[:url]` in `:unstable` channel tests and not introduce this logic.